### PR TITLE
Completed MoreEnumerables challenge

### PR DESCRIPTION
**Pull request by pdroll**

### CODE REVIEW

**Comment by pdroll** [comment]
@mikegee @jaybobo Let me know what you think!

File: challenge.rb [hunk 1]
```diff
@@ -1,26 +1,34 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map do |val|
+    val.capitalize
```
Comment (jaybobo) [inline challenge.rb]: looks good. take a look at Ruby's to_proc method...`&`. you can make this more concise.

http://www.brianstorti.com/understanding-ruby-idiom-map-with-symbol/

File: challenge.rb [hunk 2]
```diff
@@ -1,26 +1,34 @@
...
+  end
 end
 
 def fetch_the_dog(input)
```
Comment (jaybobo) [inline challenge.rb]: this works but `.select` is what we're looking for.

File: challenge.rb [hunk 3]
```diff
@@ -1,26 +1,34 @@
...
-  #implement your solution here
+  input.find_all do |animal|
+    animal.downcase == "dog"
+  end
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.dup.delete_if do |animal|
```
Comment (jaybobo) [inline challenge.rb]: this works but `.reject` is what we're looking for.

File: challenge.rb [hunk 4]
```diff
@@ -1,26 +1,34 @@
...
-  #implement your solution here
+  input.count
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  input.take(2)
 end
 
 def fetch_CD_animals(input)
```
Comment (jaybobo) [inline challenge.rb:26]: how about `.grep`?

**Comment by jaybobo** [comment]
LGTM

**Pull request closed without merging**